source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidRealTimeMultiplayerClient.cs

```diff
@@ -786,7 +786,7 @@ private class RoomUpdateCallbackProxy : AndroidJavaProxy
                 mListener.OnRoomSetupProgress(mParent.GetPercentComplete());
             }
 
-            public void onJoinedRoom( /* @OnJoinedRoomStatusCodes */ int statusCode, /* @Nullable Room */
```

Comment:
Actually, we can leave annotations in comments.

Reply:
Reverted this file.